.github/workflows/pr-linter.yml

```diff
@@ -23,26 +23,27 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Get file diff
       id: diff-files
-      uses: trilom/file-changes-action@v1.2.4
+      uses: markgilbert-git/file-changes-action@v1.2.5
+
       with:
         output: '\n'
         fileOutput: '\n'
     - name: Get Python file diff
       id: pydiff
       run: >
-        echo "::set-output name=pyfiles::$(echo -e
+        echo "pyfiles=$(echo -e
         '${{ steps.diff-files.outputs.files_added }}\n${{ steps.diff-files.outputs.files_modified }}'
```

Comment:
my pleasure.
deprecation warnings from the build system /GitHub are very high priority for me because those items obsolescing can be extinction events for a project.
The deprecation related to security issues.  Sadly, even the FOSS space needs to be more careful about that.  

I'm doing some development that leverages pynetdicom so I am inclined to tidy things up a bit.  Some of the work is just slight enhancement of existing examples.
Other bits are substantial enough that I am doing it my own public project space.
Perhaps when it matures it can go into apps.